server/api.go

```diff
@@ -382,15 +381,16 @@ func (d dexAPI) RevokeRefresh(ctx context.Context, req *api.RevokeRefreshReq) (*
 	return &api.RevokeRefreshResp{}, nil
 }
 
-func (d dexAPI) GetVehiclePrivilegeToken(ctx context.Context, req *api.GetVehiclePrivilegeTokenReq) (*api.GetVehiclePrivilegeTokenResp, error) {
-	expiry := d.serverConfig.Now().Add(time.Minute * 10) // TODO - Discuss an appropriate time
+func (d dexAPI) GetPrivilegeToken(ctx context.Context, req *api.GetPrivilegeTokenReq) (*api.GetPrivilegeTokenResp, error) {
+	expiry := d.serverConfig.IDTokensValidFor.Milliseconds() // TODO - Discuss an appropriate time
```

Comment:
`exp` is the unix timestamp for expiry. I would follow what happens in [`oauth2.go`](https://github.com/DIMO-Network/dex/blob/5706160060c12acdb51f0ea72ae94b186f62118d/server/oauth2.go#L358): take `iss`, add on `IDTokensValidFor`, then call `.Unix()`.

This is the only super serious request for a change I have.